src/core/time.d

```diff
@@ -4329,6 +4361,48 @@ string numToString(long value) @safe pure nothrow
         assert(0, "Something threw when nothing can throw.");
 }
 
+unittest
+{
+    string a = numToString(1);
+    assert(a == "1");
+
+    a = numToString(-1);
+    assert(a == "-1");
+}
+
+/++ A six digit double to string converstion function
++/
+string numToString(double value) @safe pure nothrow
+{
+    immutable negative = value < 0;
+    string result = (negative ? "-" : "") ~ numToString(cast(long)value);
+    result ~= '.';
+    result ~= numToString(cast(long)(_abs((value - cast(long)value) * 1000000)));
```

Comment:
It shouldn't have `/++` on it - just `/+` - but it _is_ used in the actual code, not just the unit tests. Both `Duration` and `MonoTime` use it in their `toString` functions, since the proper Phobos functions aren't available.